ui/src/tool-panel/location-track/dialog/location-track-validation.ts

```diff
@@ -9,15 +9,18 @@ export const ALIGNMENT_NAME_REGEX = /^[A-Za-zÄÖÅäöå0-9 \-_]+$/g;
 export const ALIGNMENT_DESCRIPTION_REGEX = /^[A-ZÄÖÅa-zäöå0-9 _\-–—+().,'"/\\<>:!?&]+$/g;
 export const ALIGNMENT_NAME_MAX_LENGTH = 50;
 
-const freeTextSchemes = [
+const MANDATORY_FREE_TEXT_SCHEMES: LocationTrackNamingScheme[] = [
     LocationTrackNamingScheme.FREE_TEXT,
     LocationTrackNamingScheme.WITHIN_OPERATING_POINT,
+] as const;
+const ALL_FREE_TEXT_SCHEMES: LocationTrackNamingScheme[] = [
     LocationTrackNamingScheme.TRACK_NUMBER_TRACK,
-];
-const specifierSchemes = [
+    ...MANDATORY_FREE_TEXT_SCHEMES,
+] as const;
+
+const MANDATORY_NAME_SPECIFIER_SCHEMES: LocationTrackNamingScheme[] = [
     LocationTrackNamingScheme.TRACK_NUMBER_TRACK,
-    LocationTrackNamingScheme.BETWEEN_OPERATING_POINTS,
-];
+] as const;
```

Comment:
GVT-3231: Validaatiota on nyt jaettu nyt osiin vähän eri tavalla. Aiemmin `freeText`:in ajateltiin olevan pakollinen aina kun se voitiin syöttää, mutta näin ei olekaan (ratanumeron raiteella se on optionaalinen.) Täten pakollisuus tarkastellaan nyt vain niiltä tyypeiltä joilla se oikeasti on pakollinen, mutta syötetyn `freeText`:in validius tarkastellaan kaikilta joille sen voi syöttää

GVT-3180: Poistettu `nameSpecifier`:in pakollisuus liikennepaikkojen välisiltä raiteilta